docs/roadmap.md

```diff
@@ -64,31 +64,31 @@ Blank -  Not available
 **WinUI 3 - Windows App SDK 1.0** is our latest release. For a detailed look at the features we're targeting in WinUI 3 in 2022, check out the [Windows App SDK feature roadmap](https://github.com/microsoft/WindowsAppSDK/blob/main/docs/roadmap.md).
 
 | Customer Capability | Project Reunion 0.5 <br/>(March 2021) | Windows App SDK 0.8 <br/>(June 2021) | Windows App SDK 1.0 <br/> (November 2021) |
```

Comment:
It's generally bad to include HTML (the br tag) in Markdown. Usually to get a soft return you can end a line with two spaces, although I'm not sure if that works in a table heading